webclient/pian/views.py

```diff
@@ -467,32 +467,31 @@ def post(self, request):
             logger.debug("pian.views.ImportovatPianView.post.label_check.unreadable_or_empty",
                          extra={"err": err})
             return HttpResponseBadRequest(_("pian.views.importovatPianView.check.unreadable_or_empty."))
-        if sheet.shape[0] == 0 or sheet.shape[1] == 0:
+        if sheet.shape[1] == 0:
             return HttpResponseBadRequest(_("pian.views.importovatPianView.check.unreadable_or_empty."))
         if sheet.shape[1] != 3:
             logger.debug("pian.views.ImportovatPianView.post.label_check.incorrect_column_count",
                          extra={"columns": sheet.columns})
-            return HttpResponseBadRequest(f'{_("pian.views.importovatPianView.check.wrongColumnName.")} '
-                                          f'{', '.join(sheet.columns)}')
```

Comment:
kdyby nahodou do budoucna, je. to treba delat napr takto:
help_translation = _('core.views.SearchListChangeColumnsView.show')
return HttpResponse(f"{help_translation} {sloupec}")